containers/Comments/Comment.js

```diff
@@ -198,7 +210,7 @@ export const Comment = (props) => {
           <Button
             icon={optionsIcon}
             type='clear'
-            onPress={() => {setCommentBeingProcessed(comment); props.toggleMenu();}}
+            onPress={() => handleOptions()}
```

Comment:
because no params passed, would `onPress={handleOptions}` work too?

Reply:
but then it passes the event from `onPress`, which clashes with the possible subcomment param of `handleOptions`

Reply:
i don't follow, the notation i suggestion would call a different function that has the same name?

Reply:
yes, handleOptions can be called from both comment and subcomment. If called from the comment we already have all the info we need, if called from the subcomment we need to tell it which subcomemnt. I think the difference between this and yours is: `onPress={handleOptions}` is equivalent to `onPress={(event)=>handleOptions(event)}`. Since the first param of `handleOptions` is supposed to be `subcomment` we would end up with the event being passed as the subcomment